src/info/manifest.c

```diff
@@ -125,6 +128,15 @@ manifestFilePack(const Manifest *const manifest, const ManifestFile *const file)
     // Flags
     uint64_t flag = 0;
```

Comment:
At first pass, I wasn't enthused about packing bits using explicit shifts - usually I prefer using bitfields in the first place. But this case it makes sense - bit fields are not portable, and the shifting only takes place in the pack and unpack routines. So ... looks good.

This  function also explains an earlier comment about using low bits for the most likely cases. It isn't because big shifts are slower (which isn't true on most architectures) , but because the varint format is more efficient when higher bits are zero. So ... the earlier comment makes sense now. 

I approve ... just being verbose in my thinking.